tasks/au/data.py

```diff
@@ -408,12 +414,30 @@ def populate_general(self):
 
             in_colnames[0] = '"{}"'.format(self._get_geoid())
 
-            cmd = 'INSERT INTO {output} ("{out_colnames}") ' \
-                  'SELECT {in_colnames} FROM {input} '.format(
-                      output=self.output().table,
-                      input=intable,
-                      in_colnames=', '.join(in_colnames),
-                      out_colnames='", "'.join(out_colnames))
+            if self.resolution in CONVERT_TO_7_DIGIT:
+                # we need 11 digit IDs (from the geography table) because
+                # the data table only has 7 digit IDs
+                in_colnames[0] = 'geo.geom_id'
+                cmd = '''
+                    INSERT INTO {output} ("{out_colnames}")
```

Comment:
Minor: as the difference is the join, extracting the common part and adding the join afterward if needed might be more clear.

Reply:
In this case I prefer keep both separated. Since `in_colnames` belongs to the common part but then is changed depending on the `resolution` and both queries have different formatted variables I think it's less prone to errors (and easier to read) if both are separated.